open_ce/docker_build.py

```diff
@@ -178,7 +177,7 @@ def build_in_container(image_name, args, arg_strings):
     _start_container(container_name)
 
     # Execute build command
-    cmd = "python {} {} {} {}".format(os.path.join(HOME_PATH, "open-ce", "open-ce", "open-ce"),
+    cmd = "source $HOME/.bashrc; python {} {} {} {}".format(os.path.join(HOME_PATH, "open_ce", "open-ce"),
```

Comment:
Do you know why this is needed? I would rather not hard code this in, since users might not have a `.bashrc` file.

Reply:
Yes. Earlier we used to copy cloned copy of open-ce code into container. But now, we are copying the files from installed package with imports like `import open_ce.utils`. To make this work inside container, PYTHONPATH needed to be exported which I did in Dockerfiles. However, at this point when python build env is invoked inside container, PYTHONPATH wasn't in effect until `source ~/.bashrc` is done.

Reply:
We are writing stuff in ~/.bashrc in our Dockerfiles for build images. So, it will always be present.

Reply:
I see. I forgot this was within the docker_build code, so we have more control over what will be there. I think this is OK then.